utils/dataset_manifest/core.py

```diff
@@ -148,18 +150,20 @@ def __iter__(self):
 class DatasetImagesReader:
     def __init__(self,
                 sources,
-                meta=None,
-                sorting_method=SortingMethod.PREDEFINED,
-                use_image_hash=False,
+                *args,
                 start = 0,
                 step = 1,
                 stop = None,
-                *args,
+                meta=None,
+                sorting_method=SortingMethod.PREDEFINED,
+                use_image_hash=False,
+                with_image_info=True,
```

Comment:
Is this argument actually supplied anywhere? I don't see it.

Reply:
It is used in tests primarily. Probably, it should be added to CLI to be really useful.

Reply:
I agree.

Reply:
The argument was removed to simplify things.